src/settings-ui/Settings.UI/ViewModels/PowerLauncherPluginViewModel.cs

```diff
@@ -170,6 +172,32 @@ public IEnumerable<PluginAdditionalOptionViewModel> AdditionalOptions
             }
         }
 
+        public IEnumerable<PluginMetadataViewModel> PluginMetadataItems => _pluginMetadataList;
+
+        /// <summary>
+        /// Generates the list of metadata properties like version, author, website from the plugin settings.
+        /// </summary>
+        /// <remarks>
+        /// The items order in the list defines the order in the ui.
+        /// </remarks>
+        private static List<PluginMetadataViewModel> GeneratePluginMetadataList(PowerLauncherPluginSettings pluginSettings)
+        {
+            List<PluginMetadataViewModel> metadataList = new()
+                {
+                    { PluginMetadataViewModel.MetadataItem(pluginSettings.Version, PluginMetadataViewModel.PluginMetadataType.Version) },
+                    { PluginMetadataViewModel.ItemSeparator() },
+                    { PluginMetadataViewModel.MetadataItem(pluginSettings.Author, PluginMetadataViewModel.PluginMetadataType.Author) },
+                };
+
+            if (Uri.IsWellFormedUriString(pluginSettings.Website, UriKind.Absolute))
+            {
+                metadataList.Add(PluginMetadataViewModel.ItemSeparator());
+                metadataList.Add(PluginMetadataViewModel.MetadataItem(pluginSettings.Website, PluginMetadataViewModel.PluginMetadataType.Link));
```

Comment:
```suggestion
                metadataList.AddRange(PluginMetadataViewModel.ItemSeparator(), PluginMetadataViewModel.MetadataItem(pluginSettings.Website, PluginMetadataViewModel.PluginMetadataType.Link));
```

Minor improvement: consider using `AddRange(...)`

Reply:
Tried it. But didn't find the right syntax. 😅